### PR TITLE
remove test wallets / new CNAME / display version

### PR DESCRIPTION
**Pull request by Kevingislason**
Some tiny tasks to prepare for deploy:
-Remove test wallets ("alice" and others)
-Change CNAME to app.dolphin.manta.network
-Display the current dolphin protocol version ("Bolo")

### CODE REVIEW

File: src/components/elements/Layouts/SidebarMenu/index.js [hunk 1]
```diff
@@ -12,7 +12,10 @@ const SidebarMenu = () => {
             <div className="logo border-white">
               <img src={Svgs.Logo} alt="logo" />
             </div>
-            <h1 className="text-3xl mb-0 pl-2 font-semibold text-accent">Dolphin</h1>
+            <div>
+              <h1 className="text-3xl mb-0 pl-2 font-semibold text-accent">Dolphin</h1>
+              <h3 className="text-l mb-0 pl-3 font-semibold text-secondary">Bolo version</h3>
```
Comment (dansofter) [inline src/components/elements/Layouts/SidebarMenu/index.js:17]: @Kevingislason Is Bolo correct? I think it was Boto?

**Review by dansofter — CHANGES_REQUESTED** [review]

File: src/components/elements/Layouts/SidebarMenu/index.js (hunk 1 above)
Comment (bhgomes) [reply src/components/elements/Layouts/SidebarMenu/index.js:17]: Yeah, should be Boto.

Comment (dansofter) [reply src/components/elements/Layouts/SidebarMenu/index.js]: Cool. Fixed the typo and committed.

**Review by dansofter — APPROVED** [review]
lgtm

**Pull request merged**